packages/app/src/runner/SpecRunner.vue

```diff
@@ -10,7 +10,7 @@
       <InlineSpecList :gql="props.gql" />
     </HideDuringScreenshot>
 
-    <HideDuringScreenshot class="w-128">
+    <HideDuringScreenshot class="min-w-320px">
```

Comment:
(Just noting that this is a temporary measure until drag to resize gets implemented)